docs/dashboards/dashboards-migration/index.md

```diff
@@ -9,10 +9,12 @@ The **Migrate to Dashboards** functionality allows you to transform **Dashboards
 
 ## Advantages
 
-**Dashboards**
-
-- Persistent release and update with new feature additions.
-- Provides the unique ability to analyze metrics metadata and log data on the same dashboard in an integrated seamless view. This is what you need to monitor and manage a Kubernetes environment effectively.
-- You can control the visual display of metric and log data.
-- Streamline dashboard configuration and on-the-fly analytic visualizations with its new templating features.
+- Visulize queries with new dashboard panels, including honeycombs, bubble charts, heat maps, and funnel charts.
```

Comment:
Visualize has a typo here. Can you also link to  https://help.sumologic.com/docs/dashboards/panels/

Reply:
Thanks @oshoghi. I have fixed your comment.